app/views/foods/_form.html.erb

```diff
@@ -1,25 +1,25 @@
 
 <%= form_with(model: food, html: { class: "" }) do |form| %>
 <%= render "shared/error_form_message", resource: food %>
-  <div class="field">
-    <%= form.label "Food name", style: "display: block" %>
-    <%= form.text_field :name, required: true %>
-  </div>
+<div class="field">
```

Comment:
- [ ] Also, right here!

![Screenshot (1)7](https://user-images.githubusercontent.com/47625407/219825460-d6cc881d-ed7e-4b9a-baf5-ac7bbd374235.png)

Reply:
Hey @MahmoudBakr23, I cannot see the duplicate sign-up on my end, can I submit it again?